clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1409,4 +1409,55 @@ public boolean visible(String name, Map<String, Object> parsedConfig) {
         };
     }
 
+    public String toHtmlList() {
+        return toHtmlList(Collections.<String, String>emptyMap());
+    }
+
+    /**
+     * Converts this config into an HTML list that can be embedded into docs.
+     * If <code>dynamicUpdateModes</code> is non-empty, a "Dynamic Update Mode" label
+     * will be included in the config details with the value of the update mode. Default
+     * mode is "read-only".
+     * @param config The config to convert
+     * @param dynamicUpdateModes Config name -&gt; update mode mapping
+     */
+    public String toHtmlList(Map<String, String> dynamicUpdateModes) {
```

Comment:
Can we call this `toHtml` to go along with the generically named `toRst`? If we want to change the output in the future, we won't need to add a new API.

Reply:
I agree, done!